portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -163,5 +173,5 @@ const addSubmissionData = async (dealId, existingApplication) => {
 };
 
 module.exports = {
-  addSubmissionData, submissionPortalActivity, submissionTypeToConstant, getUserInfo,
+  addSubmissionData, submissionPortalActivity, submissionTypeToConstant, getUserInfo, addSubmissionDateToIssuedFacilities,
```

Comment:
Convert to multiple lines please to improve readability.

Reply:
done